xarray/test/test_dask.py

```diff
@@ -300,3 +300,31 @@ def test_stack(self):
                              dims=['w', 'z'])
         assert stacked.data.chunks == expected.data.chunks
         self.assertLazyAndIdentical(expected, stacked)
+
+    def test_dot(self):
```

Comment:
The reason to make this test something simpler is reduce the redundancy with the other `test_dot` test case. Complex tests can be good to make sure things work, but it's often easier to debug simpler/shorter ones.